charts/backend-service/templates/routes.yaml

```diff
@@ -134,7 +134,7 @@ spec:
       {{- range .Values.routes.paths.scoped.allowList }}
         - path:
             type: PathPrefix
-            value: {{ default "/scoped" $.Values.routes.paths.scoped.pathOverride }}{{ . }}
+            value: {{ default "/scoped" $.Values.routes.paths.scoped.pathOverride }}{{ include "backendService.routePrefix" $ }}{{ . }}
```

Comment:
I think the `{{ . }}` needs to be part of the URL Rewrite as well or it will be removed.
Not tested, but my assumption is, that the URLRewrite takes the full recognized Prefix and replaces it.